openstack/mrs/v1/cluster/results.go

```diff
@@ -72,24 +89,27 @@ type CreateResult struct {
 	golangsdk.Result
 }
 
-func (r CreateResult) Extract() (*ClusterResult, error) {
-	var s ClusterResult
-	err := r.ExtractInto(&s)
-	return &s, err
+func (r CreateResult) Extract() (*CreateClusterResult, error) {
+	var s CreateClusterResult
+	err := r.ExtractIntoStructPtr(&s, "")
+	if err != nil {
+		return nil, err
+	}
+	return &s, nil
 }
 
 type GetResult struct {
 	golangsdk.Result
 }
 
 func (r GetResult) Extract() (*Cluster, error) {
-	var s Cluster
-	err := r.ExtractInto(&s)
-	return &s, err
-}
+	s := new(Cluster)
+	err := r.ExtractIntoStructPtr(s, "cluster")
+	if err != nil {
+		return nil, err
+	}
+	return s, nil
```

Comment:
Please use the same structure initialization approach in both functions :)

Reply:
done